src/PullCountStackTooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChartTooltipProps} from '@patternfly/react-charts';
+import * as _ from 'lodash';
+import React from 'react';
+import './PullCountStackTooltip.css';
+import { Timespan } from './Timespan';
+
+interface IPullCountTooltipProps extends ChartTooltipProps {
+  timespan: Timespan;
+  bars: {[tag: string]: Array<{name: string, x: number, y: number}>};
+}
+
+interface IPullCountTooltipState {}
+
+interface IToolTipItem {
+  name: string;
+  x: number;
+  y: number;
+}
+
+export class PullCountStackTooltip extends React.Component<IPullCountTooltipProps, IPullCountTooltipState> {
+  constructor(props: IPullCountTooltipProps) {
```

Comment:
I think you can remove this constructor.

Reply:
True, removed.